client/src/pages/WeeklyTable.js

```diff
@@ -61,13 +72,20 @@ const WeeklyTable = ({ bookings, rowsCount, refreshBooking }) => {
 						<th></th>
 						{week.map((date) => (
 							<th key={date}>
-								{formatBookingDate(date)} <ModalBookingScreen bookingDate={date} refreshBooking={refreshBooking} />
+								{formatBookingDate(date)}{" "}
+								<ModalBookingScreen
+									bookingDate={date}
+									refreshBooking={refreshBooking}
+								/>
+								<br />
+								{getBookingByDay(bookings, date) === 5
```

Comment:
the 5 would be better as a function

-> const maxDesksForDay = 5 -> then when it changes it will change in both places (=== 5 and "/5")